### ISSUE TITLE
Support recent OCR-D code

### ISSUE THREAD

**Issue by stweil** [OCR-D/quiver-benchmarks#23]
QuiVer benchmarks is currently based on a rather old `ocrd/all:2023-03-26`.

It would be good to run benchmarks with recent versions of OCR-D, but the code contains assumptions which prevent that:

- It expects models in /usr/local/share.
- Others?

**Comment by stweil** [comment]
@mweidling, are there plans to update the code to support latest OCR-D? Or did you stop the work on QuiVer benchmarks?

**Comment by mweidling** [comment]
@stweil Yes, there is. I'm currently working on it in the `update-ocrd_all` branch.
